ajax/libs/rambda/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "rambda",
+  "filename": "webVersion.js",
+  "version": "0.5.1",
+  "description": "Lightweight alternative to Ramda",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/selfrefactor/rambda.git"
+  },
+  "keywords": [
+    "ramda",
+    "fp",
+    "functional",
+    "utility"
+  ],
+  "author": {
+    "name": "self_refactor",
+    "url": "https://github.com/selfrefactor"
+  },
+  "license": "MIT",
```

Comment:
@hare1039 I think you should notify author that the license should be changed to `MIT` because the `package.json` file on both GitHub repo and NPM shows `ISC`. Please double confirm this. Thank you.